src/app.js

```diff
@@ -1 +1,126 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from 'axios';
+
+async function fetchData() {
+    try {
+        const response = await axios.get('https://restcountries.com/v2/all');
+        const countries = response.data
+        // console.log(response.data);
+
+
+        countries.sort((a, b) => {
+            return a.population - b.population;
+        });
+
+
+        countryItem(countries);
+    } catch (e) {
+        console.error(e);
+    }
+}
+
+fetchData();
+
+function countryItem(countries) {
+
+    const countryList = document.getElementById('country-list');
+    const allCountries = countries.map((country) => {
```

Comment:
Je zou hier nog spreading kunnen toepassen, zodat je niet overal `country.` vóór hoeft te zetten
```suggestion
    const allCountries = countries.map(({ name, flags, region, population }) => {
```